include/libdnf/base/goal_elements.hpp

```diff
@@ -87,6 +87,7 @@ enum class GoalAction {
     DISTRO_SYNC_ALL,
     DOWNGRADE,
     REMOVE,
+    PROVIDES_INSTALL,
```

Comment:
Why a plural? What about `PROVIDE_INSTALL` or `INSTALL_PROVIDE`? What about to ask team?

Reply:
Well, I followed the libsolv flag name - `SOLVER_SOLVABLE_PROVIDES`. But I don't have any problem renaming them to singular.

Reply:
OK, changed to INSTALL_VIA_PROVIDE.